docs/sources/flow/reference/components/otelcol.receiver.opencensus.md

```diff
@@ -0,0 +1,226 @@
+An empty list means that CORS is not enabled at all. A wildcard (*) can be
+used to match any origin or one or more characters of an origin.
+
+## Blocks
+
+The following blocks are supported inside the definition of
+`otelcol.receiver.opencensus`:
+
+Hierarchy | Block | Description | Required
+--------- | ----- | ----------- | --------
+grpc | [grpc][] | Configures the gRPC/HTTP server to receive telemetry data. | no
+grpc > tls | [tls][] | Configures TLS for the gRPC server. | no
+grpc > keepalive | [keepalive][] | Configures keepalive settings for the configured server. | no
+grpc > keepalive > server_parameters | [server_parameters][] | Server parameters used to configure keepalive settings. | no
+grpc > keepalive > enforcement_policy | [enforcement_policy][] | Enforcement policy for keepalive settings. | no
+output | [output][] | Configures where to send received telemetry data. | yes
+
+The `>` symbol indicates deeper levels of nesting. For example, `grpc > tls`
+refers to a `tls` block defined inside a `grpc` block.
+
+[grpc]: #grpc-block
+[tls]: #tls-block
+[keepalive]: #keepalive-block
+[server_parameters]: #server_parameters-block
+[enforcement_policy]: #enforcement_policy-block
+[output]: #output-block
+
+### grpc block
+
+The `grpc` block configures the gRPC/HTTP server used by the component. If the
+`grpc` block isn't provided, a server will be started using default parameters.
+
+The following arguments are supported:
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`endpoint` | `string` | `host:port` to listen for traffic on. | `"0.0.0.0:4317"` | no
+`transport` | `string` | Transport to use for the gRPC server. | `"tcp"` | no
+`max_recv_msg_size` | `string` | Maximum size of messages the server will accept. 0 disables a limit. | | no
+`max_concurrent_streams` | `number` | Limit the number of concurrent streaming RPC calls. | | no
+`read_buffer_size` | `string` | Size of the read buffer the gRPC server will use for reading from clients. | `"512KiB"` | no
+`write_buffer_size` | `string` | Size of the write buffer the gRPC server will use for writing to clients. | | no
+`include_metadata` | `boolean` | Propagate incoming connection metadata to downstream consumers. | | no
+
+In order to use plain HTTP/JSON, specify the `endpoint` attribute. There is no HTTP-specific block.
+The HTTP/JSON address is the same as gRPC as the protocol is recognized and processed accordingly.
+
+To write traces with HTTP/JSON, `POST` to `[address]/v1/trace`. The JSON message format parallels the gRPC protobuf format, see this [OpenApi spec for it](https://github.com/census-instrumentation/opencensus-proto/blob/master/gen-openapi/opencensus/proto/agent/trace/v1/trace_service.swagger.json).
+
+Note that `max_recv_msg_size`, `read_buffer_size` and `write_buffer_size` are formatted in a special way, 
+so that the units are included in the string - e.g. "512KiB" or "1024KB".
```

Comment:
```suggestion
Note that `max_recv_msg_size`, `read_buffer_size` and `write_buffer_size` are formatted in a special way, 
so that the units are included in the string, e.g., "512KiB" or "1024KB".
```